documentation/views/control.rst

```diff
@@ -1,34 +1,33 @@
 .. _control:
 
 *****************
-Flexibility actions
+Flexibility opprtunities
```

Comment:
A lot of underdefined/ambiguous terminology in this section. I propose we discuss this thoroughly in case we activate this UI view.

Reply:
Just a note: this page is currently visible in the documentation. I suggest to move the content which does not talk about the UI into a new page which deals with concepts. That PR for #45  will land soon in your queue. There we might give it another look.